sample-apps/blank-java/src/test/java/example/InvokeTest.java

```diff
@@ -29,30 +22,34 @@
 import com.amazonaws.xray.strategy.sampling.NoSamplingStrategy;
 
 class InvokeTest {
-  private static final Logger logger = LoggerFactory.getLogger(InvokeTest.class);
-  Gson gson = new GsonBuilder()
-          .registerTypeAdapter(SQSEvent.class, new SQSEventDeserializer())
-          .setPrettyPrinting()
-          .create();
 
   public InvokeTest() {
-    AWSXRayRecorderBuilder builder = AWSXRayRecorderBuilder.standard();
-    builder.withSamplingStrategy(new NoSamplingStrategy());
-    AWSXRay.setGlobalRecorder(builder.build());
+      AWSXRayRecorderBuilder builder = AWSXRayRecorderBuilder.standard();
+      builder.withSamplingStrategy(new NoSamplingStrategy());
+      AWSXRay.setGlobalRecorder(builder.build());
   }
 
   @Test
   void invokeTest() {
-    AWSXRay.beginSegment("blank-java-test");
-    String path = "src/test/resources/event.json";
-    String eventString = loadJsonFile(path);
-    SQSEvent event = gson.fromJson(eventString, SQSEvent.class);
-    Context context = new TestContext();
-    String requestId = context.getAwsRequestId();
-    Handler handler = new Handler();
-    String result = handler.handleRequest(event, context);
-    assertTrue(result.contains("totalCodeSize"));
-    AWSXRay.endSegment();
+      AWSXRay.beginSegment("blank-java-test");
+      String path = "src/test/resources/event.json";
+      String eventString = loadJsonFile(path);
```

Comment:
Use the to ensure you are correctly serialising the event json. Lambda uses a custom ObjectMapper. Your example will cause problems for some event types.

https://github.com/aws/aws-lambda-java-libs/tree/main/aws-lambda-java-tests